success-criteria.md

```diff
@@ -417,6 +393,10 @@ stakeholder group.
             -   Access to adult content by an appropriate guardian to prevent
                 unauthorized viewing by underage family members
 
+            -   Tracking that would record any behavioral or user data of those under the age of 13
+
+            -   The joining of data from a web interaction or session with externally collected data.
```

Comment:
The considerations of a user here do not include considerations of the consequences of behind-the-scenes data joining. We should include clear transparency around this concept along with tools of remedy.